config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -53,8 +53,17 @@ config SYSTEM_WORKQUEUE_STACK_SIZE
     default 2560 if CHIP_WIFI
     default 2048 if CHIP_ICD_LIT_SUPPORT
 
+if CHIP_WIFI
+
 config HEAP_MEM_POOL_SIZE
-    default 110720 if CHIP_WIFI
+    default 48336 if BUILD_WITH_TFM
+    default 40144
+
+config NRF_WIFI_DATA_HEAP_SIZE
+    default 50576 if BUILD_WITH_TFM
```

Comment:
Where do these values come from? The description says "redistribute", but the given sizes do not sum up to 110720. Are you sure it's safe to decrease the pool size?

Reply:
@rado17 ?

Reply:
Need to correct the `NRF_WIFI_DATA_HEAP_SIZE`. The values are interchanged. With `BUILD_WITH_TFM` where `MBEDTLS_HEAP` will be disabled, we need to reduce the `NRF_WIFI_DATA_HEAP_SIZE` by 8192 bytes (to 42384).  Pushed the changes now.
`NRF_WIFI_CTRL_HEAP_SIZE` is set to 20000 by default. The total comes to 110720.

Reply:
We have done some basic performance tests with `NRF_WIFI_DATA_HEAP_SIZE` set to 40000 and `NRF_WIFI_CTRL_HEAP_SIZE` set to 20000 and didn't see any issues. Reserving 40144 bytes for K_HEAP is required for supplicant and socketpair subsystems.